e2e/cypress/integration/channel/message_spec.js

```diff
@@ -140,4 +141,63 @@ describe('Message', () => {
             cy.get(divPostId).should('have.class', 'post--system');
         });
     });
+
+    it('MM-16730 Reply to an older message', () => {
+        cy.getCurrentChannelId().then((channelId) => {
+            // # Get yesterdays date in UTC
+            const yesterdaysDate = new Date().getTime() - 86400000;
+
+            // # Post a day old message
+            cy.postMessageAs({sender: sysadmin, message: 'Hello from yesterday', channelId, createAt: yesterdaysDate}).
+                its('id').
+                should('exist').
+                as('yesterdaysPost');
+        });
+
+        // # Add two subsequent posts
+        cy.postMessage('One');
+        cy.postMessage('Two');
+
+        cy.get('@yesterdaysPost').then((postId) => {
+            // # Open RHS comment menu
+            cy.clickPostCommentIcon(postId);
```

Comment:
Depending on the volume of messages, this step fails for me because "yesterdays" post is pushed up beyond where it's visible. Tacking on some `scrollIntoViews()` where appropriate should address this.

Reply:
Nice catch. I'll add that in.

My understanding is that it's only needed before the `clickPostCommentIcon` since there are no actions on the other posts in the center screen. Everything should remain in the DOM.

Let me know if my understanding is specious.

Reply:
Eventually, we will be parallelizing these tests so we want to guard against other tests possibly writing to this same channel, what you did seems perfectly fine.